helpers/StateContractMigrationHelper.ts

```diff
@@ -141,13 +141,27 @@ export class StateContractMigrationHelper extends ContractMigrationSteps {
   @log
   async upgradeContract(
     stateContract: Contract,
-    redeployVerifier = true,
-    redeployCrossChainProofValidator = true,
+    opts: {
+      redeployVerifier: boolean;
```

Comment:
`redeployGroth16Verifier` is better to differentiate

Reply:
Yes, sure.